app/views/hiring_staff/vacancies/_show.html.haml

```diff
@@ -35,60 +35,21 @@
         %span.govuk-caption-l
           = @vacancy.location
 
-      %h2.govuk-heading-m= t('jobs.description')
-      %p= @vacancy.job_description
+      .govuk-tabs{"data-module": "govuk-tabs"}
+        %h2.govuk-tabs__title
+          = t('jobs.key_information')
+        %ul.govuk-tabs__list
+          %li.govuk-tabs__list-item.govuk-tabs__list-item--selected
+            %a.govuk-tabs__tab.tab-links-override{:href => "#job-details"}
+              = t('jobs.job_details')
+          %li.govuk-tabs__list-item
+            %a.govuk-tabs__tab.tab-links-override{:href => "#school-overview"}
```

Comment:
Does rubocop alert you `Style/HashSyntax: Use the new Ruby 1.9 hash syntax`? Not sure if we have linting on these haml files.

Reply:
Nope it didn't, but I can change this file to use the newer syntax 😄